README.md

```diff
@@ -1,17 +1,15 @@
+**Fork Awesome is a suite of 796 pictographic and brand icons for easy, scalable vector graphics on websites and beyond.**
 
-Following concerns regarding [the development of Font Awesome](https://github.com/FortAwesome/Font-Awesome/issues/12199#issuecomment-362919956), the PR Freeze since Oct 2016 and the direction [Fort Awesome](https://fortawesome.com/) is taking with the version 5.0 of their project, we are forking Font Awesome (4.7), in order to build on this amazing tool Dave Gandy has given us, while at the same time allowing this project to be run by a distributed community of contributors.
+This project, as the name suggests, is a fork of [Font Awesome](https://fontawesome.com). Font Awesome was originally created by [Dave Gandy](https://twitter.com/davegandy) as a community project. However, as Font Awesome developed, PRs from the community stopped being accepted (Oct. 2016) and as of version 5.0 [the build system became private](https://github.com/FortAwesome/Font-Awesome/issues/12199#issuecomment-362919956) (Feb. 2018).
 
-A tremendous gratitude is given to the whole team behind [Font Awesome](https://fontawesome.com), and you are encouraged to support them and buy Font Awesome Pro for your project.
+With gratitude to Dave Gandy and the Font Awesome team, [Julien](https://github.com/xuv) [Deswaef](@xuv@merveilles.town) forked Font Awesome 4.7 into Fork Awesome 1.0 in [Feb. 2018](https://github.com/ForkAwesome/Fork-Awesome/releases/tag/1.0.0), to continue building on the amazing resource in a fully reproduceable way, as a distributed community of contributors.
 
-Though If you believe in distributed open source design and wish to be part of this new adventure, please start submitting patches and suggestions of improvement.
-
-Also, if you care about owning the build process of your icon font, this project will give you that. The whole pipeline is shared and free to use for any purpose. You could be spinning your own icon font with your clever name by just forking this project and changing a few settings.
+After a period of extended inactivity (i.e. no release since [Feb. 2019](https://github.com/ForkAwesome/Fork-Awesome/releases)), several Fork Awesome users [got](https://github.com/ForkAwesome/Fork-Awesome/issues/292) [together](https://github.com/ForkAwesome/Fork-Awesome/issues/235) to try and revive the project 🌱 Julien, after initiating and faithfully maintaining the project, was unable to continue, and hadn't found anyone to take over. Now we have the necessary access to go forward, catch up with backlogged issues and PR's, update technology and documentation — even reevaluate the project aims as icon fonts gradually [become](https://www.irigoyen.dev/blog/2021/02/17/stop-using-icon-fonts/) [deprecated](https://cloudfour.com/thinks/seriously-dont-use-icon-fonts/). Come say hi in the [#forkawesome:matrix.org](https://app.element.io/#/room/#forkawesome:matrix.org) matrix room 🙂
```

Comment:
- Perhaps, there needs to be a line break before 🌱 ? :thinking: 
- _faithfully maintaining the project **for a year**_
- hadn't => did not
- There are too many sentence joins with commas
- PR's => pull requests
- The matrix link is still here

Reply:
- Don't you you mean, after :seedling: ?
- I think we can leave contribution time ambiguous... he was contributing longer than a year
- 'Did not' seems clearer :+1: 
- I do like commas! But perhaps you're right. I can try and split it up into more sentences. Like this ;p
- PR -> pull request :+1: 
- Yes, link still there intentionally. I think it's good to have that quite present, especially after I talked to some others.

Reply:
I do not feel too strongly about the matrix link, but if you are going to keep it, please change the URL to use matrix.to.